### DISCUSSION TITLE
关于“下载”功能的一些建议

### DISCUSSION THREAD

**Discussion by DeepChirp** [the1812/Bilibili-Evolved#1540] [功能建议 / Ideas]
①有些视频在DASH格式下只提供H264编码，而没有H265，在这种情况下能否提醒用户呢？或是直接将H265选项变灰？
实例：https://www.bilibili.com/video/av59637872
②DASH格式下能否在“导出”选项中添加一个“ffmpeg-合并”选项，其中的文件名为原始的名称呢？使用IDM下载之后，名字乱七八糟的，很多文件混在一起，不太方便寻找哪些文件是对应的。特别是多个下载任务同时进行时。
③音频下载能否提供多种音质的下载呢？这个可能比较麻烦，好像网页端没有提供音质选择，但是手机客户端缓存时的确可以选择音质的。
谢谢！

**Comment by the1812** [comment]
1. 那里写的是`首选编码`, 意思就是有的话就用这个编码, 没有就默认降级, 你喜欢H265就一直开着H265就行了; 如果需要判断有没有H265, 切到H264看一下预计大小有没有变化, 变化了说明是有H265的
2. 短期内建议换用 aria2 下载, 下载视频相关功能这段时间不会再添加新功能
3. 网页端确实没有, 将来有了我再加上
